src/code/data/migrations/migration-mixin.coffee

```diff
@@ -17,3 +22,12 @@ module.exports =
     else
       log.info "  skipped : #{@name()}"
     data
+
+  # Change x.y to "x.y.0". The only annoyance is we have to special-case 1.95,
+  # as this was supposed to be < 1.10.0
+  _semverize: (v) ->
```

Comment:
This looks like it handles converting the legacy floating point version numbers. I was worried about it, but it looks good.  Have you smoke tested against some saved models? Thanks for taking care of this.